torch_xla/csrc/tensor.h

```diff
@@ -267,6 +267,18 @@ class XLATensor {
   static XLATensor narrow(const XLATensor& input, xla::int64 dim,
                           xla::int64 start, xla::int64 length);
 
+  // Takes a slice from the input as R1 at the specified offset and reshapes it
+  // into the provided size.
+  static XLATensor as_strided(
+      const XLATensor& input,
+      tensorflow::gtl::ArraySlice<const xla::int64> size,
```

Comment:
There is written nowhere this has to be array slices.
These APIs has only one call site, which is already creating a move-able vector.
I'd just pass that through down to the op.

Reply:
Done.